dip.yml

```diff
@@ -0,0 +1,90 @@
+version: "7.1"
+
+# Define default environment variables to pass
+# to Docker Compose
+environment:
+  RAILS_ENV: development
+
+compose:
+  files:
+    - .dockerdev/compose.yml
+  project_name: trucking_app
+
+interaction:
+  # Run a Rails container without any dependent services (useful for non-Rails scripts)
+  bash:
+    description: Run an arbitrary script within a container (or open a shell without deps)
+    service: rails
+    command: /bin/bash
+    compose_run_options: [no-deps]
+
+  # A shortcut to run Bundler commands
+  bundle:
+    description: Run Bundler commands
+    service: rails
+    command: bundle
+    compose_run_options: [no-deps]
+
+  yarn:
+    description: Run Yarn commands
+    service: rails
+    command: yarn
+    compose_run_options: [no-deps]
+
+  rake:
+    description: Run Rake commands
+    service: runner
+    command: bundle exec rake
+
+  rails:
+    description: Run Rails commands
+    service: rails
+    command: bundle exec rails
+    subcommands:
+      s:
+        description: Run Rails server at http://localhost:3000
+        service: web
+        compose:
+          run_options: [service-ports, use-aliases]
```

Comment:
```suggestion
  rails:
    description: Run Rails commands
    service: runner
    command: bundle exec rails
    subcommands:
      s:
        description: Run Rails server at http://localhost:3000
        service: rails
        compose:
          run_options: [service-ports, use-aliases]
```

Reply:
![image](https://user-images.githubusercontent.com/70818761/175536283-37972ed3-2364-4a74-b977-d9cde378fda0.png)
Everything is working fine on my machine
(Added chmod in makefile so we can remove old webpacker packs)